src/components/Citation.vue

```diff
@@ -13,33 +11,32 @@ export default defineComponent({
     sep(): string {
       return ". "
     },
-    date(): string {
-      const d = this.citation.date
-      return d === undefined ? "" : " (" + formatter.format(d) + ")"
-    },
     href(): string {
       return this.citation.url.toString()
     },
     title(): string {
-      const t = this.citation.title
-      return t === undefined ? this.citation.url.toString() : t
+      const title = this.citation.title
+      return title ? title : this.citation.url.toString()
```

Comment:
Da könnte man auch kürzer
`return title | this.citation.url.toString()` schreiben.